### PR TITLE
feature: PlayerItem.getMeta()

### PR DESCRIPTION
**Pull request by corporateanon**

### CODE REVIEW

File: src/lib/index.ts [hunk 1]
```diff
@@ -3,6 +3,59 @@ import { MPlayerManager } from './mplayerManager'
 const DEFAULT_OP_TIMEOUT = 2000;
 const OPEN_OP_TIMEOUT = 8000;
 
+const parseKeyValueList = (data: string): { [id: string]: string } => {
```
Comment (cdunford) [inline src/lib/index.ts]: I would prefer to see these functions exported in a separate file just to keep things cleaner. This file is already quite large and could be split up a bit better.

File: src/lib/index.ts [hunk 2]
```diff
@@ -65,6 +118,26 @@ export class MPlayerMediaItem {
     }, DEFAULT_OP_TIMEOUT);
   }
 
+  /**
+   * getMeta
+   * 
+   * @returns a promise that resolves with the metadata
+   */
+  public getMeta(): Promise<Metadata> {
```
Comment (cdunford) [inline src/lib/index.ts]: This is on the public interface - I would prefer the name to be `getMetadata()`

File: src/lib/index.ts (hunk 1 above)
Comment (cdunford) [reply src/lib/index.ts]: Additionally, please add some jsdoc comments on the new functions.

**Review by cdunford — CHANGES_REQUESTED** [review]
Please also add some tests which cover the new public function.

Thanks for submitting this PR!

**Comment by corporateanon** [comment]
Thanks for the code review. I will implement   all changes next week.

**Comment by corporateanon** [comment]
@cdunford Ignore this PR, it is to my own repository. This one is correct: https://github.com/cdunford/mplayer-as-promised/pull/1

File: src/lib/index.ts (hunk 2 above)
Comment (corporateanon) [reply src/lib/index.ts]: :+1:

File: src/lib/index.ts (hunk 1 above)
Comment (corporateanon) [reply src/lib/index.ts]: :+1:

**Pull request merged**